spec/index.bs

```diff
@@ -1601,20 +1601,20 @@ interface GPUAdapter {
 
 <script type=idl>
 dictionary GPUDeviceDescriptor : GPUObjectDescriptorBase {
-    sequence<GPUFeatureName> nonGuaranteedFeatures = [];
-    record<DOMString, GPUSize32> nonGuaranteedLimits = {};
+    sequence<GPUFeatureName> nonUniversalFeatures = [];
```

Comment:
since we are back at the hardest problem of computer science, can we call them `requestedFeatures` and `requestedLimits`? the "nonUniversal" sounds very opaque to me :)

Reply:
Hardest problem indeed ^^! What about `features` and `limits`? I lost track of why this wasn't good enough.

Reply:
We're trying to get across the point that if you do this, you will lose compatibility with some devices. We initially thought to require a boolean flag `allowNonGuaranteedCapabilities = true` in order to do this but decided to just put it in these names.

Reply:
IDK it feels that something that can be solved through educating developers instead of trying to find the perfect name. It's not clear that one exists given how hard we thought about it before finding `nonGuaranteed` that turned out to still have issues..